### PR TITLE
Support Borderless Fullscreen

### PR DESCRIPTION
**Pull request by ReillyBrogan**
Removes "Windowed" checkbox and replaces it with a "Window Mode" combobox with "Window", "Exclusive Fullscreen", and "Borderless Fullscreen" as options. "Window" and "Exclusive Fullscreen" behave identically to the previous "Windowed" and not-windowed options respectively, but "Borderless Fullscreen" launches the game with the `-popupwindow` flag which causes Unity to launch in borderless fullscreen mode. The default is Window which launches the previous default.

I tried to keep the existing code conventions but this is my first time working with C# so my changes might not be idiomatic. I'm also unsure if the French and Russian translations are correct, I just plugged the english words into Google Translate and hoped for the best.

Tested under Borderless Fullscreen:
* Going in-game
* Battles
* Cutscenes
* World Map

Not tested:
* Tetra master and other minigames

Also tested working with Moguri.

IMO there's no reason to think that BF would have any issues that don't occur in the previous Window mode. 

Screenshot of launcher:
image

### CODE REVIEW

**Comment by ReillyBrogan** [comment]
(pretty sure "Debuggable" was cut off even before I made changes)

**Comment by ReillyBrogan** [comment]
@Albeoris @snouz Any chance I could get this reviewed? Also, I see there is a separate branch for Moguri, do I need to add a separate PR to that branch to get my changes integrated into a future version of Moguri?

File: Memoria.Launcher/Languages/ru.xml [hunk 1]
```diff
@@ -32,7 +32,10 @@
 	<Settings
 			ActiveMonitor = "Активный монитор:"
 			Resolution = "Разрешение:"
-			Windowed = "В окне"
+			WindowMode = "Оконный режим"
```
Comment (Albeoris) [inline Memoria.Launcher/Languages/ru.xml:35]: Режим экрана

File: Memoria.Launcher/Languages/ru.xml [hunk 2]
```diff
@@ -32,7 +32,10 @@
...
+			Window = "Окно"
+			ExclusiveFullscreen = "полноэкранный"
```
Comment (Albeoris) [inline Memoria.Launcher/Languages/ru.xml:37]: На весь экран

File: Memoria.Launcher/Languages/ru.xml [hunk 3]
```diff
@@ -32,7 +32,10 @@
...
+			BorderlessFullscreen = "полноэкранный режим без полей"
```
Comment (Albeoris) [inline Memoria.Launcher/Languages/ru.xml:38]: Без рамок

**Review by Albeoris — APPROVED** [review]
Good job!

**Comment by Albeoris** [comment]
> @Albeoris @snouz Any chance I could get this reviewed? Also, I see there is a separate branch for Moguri, do I need to add a separate PR to that branch to get my changes integrated into a future version of Moguri?

Nope. It's main repository.

**Pull request merged**